t/app/controller/report_new.t

```diff
@@ -2185,4 +2185,33 @@ subtest "extra google analytics code displayed on email confirmation problem cre
     };
 };
 
+my $he = Test::MockModule->new('HighwaysEngland');
```

Comment:
Tests are fine, but I don't think report_new.t is the right file (plus it's long enough!), it's testing the geocoder on the around page, so around.t I'd say? (Or if the test was lower level calling the function in UK.pm directly or HE directly, a uk cobrand file, or geocoder test file, etc)